webapi/errors.go

```diff
@@ -107,6 +110,8 @@ func (e apiError) defaultMessage() string {
 		return "fee transaction could not be broadcast"
 	case errCannotBroadcastFeeUnknownOutputs:
 		return "fee transaction could not be broadcast due to unknown outputs"
+	case errInvalidTimestamp:
+		return "request timestamp is not newer than all previous requests"
```

Comment:
s / request timestamp is not newer than all previous requests / request timestamp is older than the last request

Reply:
Hm, your suggestion isnt quite accurate, but I take your point. I will update the message